experimental/packages/opentelemetry-sdk-node/test/TracerProviderWithEnvExporter.test.ts

```diff
@@ -1,313 +0,0 @@
-/*
- * Copyright The OpenTelemetry Authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import { diag } from '@opentelemetry/api';
-import {
-  ConsoleSpanExporter,
-  SimpleSpanProcessor,
-  BatchSpanProcessor,
-} from '@opentelemetry/sdk-trace-base';
-import * as assert from 'assert';
-import * as Sinon from 'sinon';
-import { env } from 'process';
-import {
-  OTLPTraceExporter as OTLPProtoTraceExporter,
-  OTLPTraceExporter,
-} from '@opentelemetry/exporter-trace-otlp-proto';
-import { OTLPTraceExporter as OTLPHttpTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
-import { OTLPTraceExporter as OTLPGrpcTraceExporter } from '@opentelemetry/exporter-trace-otlp-grpc';
-import { ZipkinExporter } from '@opentelemetry/exporter-zipkin';
-import { JaegerExporter } from '@opentelemetry/exporter-jaeger';
-import { TracerProviderWithEnvExporters } from '../src/TracerProviderWithEnvExporter';
-
-describe('set up trace exporter with env exporters', () => {
```

Comment:
would it be possible to re-purpose some of the tests from this file? 
I see that the new utils have some lines that are not covered by tests. 🤔

Reply:
I'll do another review and put the selected tests into the sdk tests